blueprints/base/services.go

```diff
@@ -24,7 +24,7 @@ var (
 )
 
 var (
-	IOReader           = &cg.GoRef{PkgPath: "io", Name: "Reader"}
+	IOReader           = &cg.GoRef{PkgPath: "io", Name: "ReadCloser"}
```

Comment:
```suggestion
	IOReadCloser           = &cg.GoRef{PkgPath: "io", Name: "ReadCloser"}
```

Reply:
and also update https://github.com/ipld/edelweiss/blob/main/blueprints/services/dagjson-over-http/client.go#L58